### DISCUSSION TITLE
Thank you for restoring HTML and text in github pages. Now my screen reader can read github pages again.

### DISCUSSION THREAD

**Discussion by superkuh** [community/community#109376] [Accessibility]
### Select Topic Area

Product Feedback

### Body

I noticed that github has restored HTML pages with actual text and links in them as the default behavior (as opposed to only being there if the javascript web application was successfully executed). I suppose this is some server side rendering. Regardless, after about a year of not being able to use the github website due to there being no text for my screenreader now it has text to read again. Thank you! I figured I'd mention this since usually you only get negative feedback.

**Comment by queenofcorgis** [comment]
We're so happy to hear this @superkuh - we appreciate all feedback both positive and negative, but were glad this restoration has made the website accessible for you!

**Comment by superkuh** [comment]
Unfortunately it looks like github has been changed again and the previous accessibility for people with screen readers has been removed again. Now text does not render in except the first "post" of whatever page.

Dang. I take back what I said.
